src/bridge/IrcHandler.ts

```diff
@@ -1003,6 +1005,7 @@ export class IrcHandler {
             // Probably the bot
             return undefined;
         }
+        const userId = client.userId;
```

Comment:
I assume this is just for readability, and not because `client` will ever change. Or do we worry about it getting modified externally during an await?

Reply:
Typescript didn't like using client.userId, because it can be undefined and we had a arrow function further down which needed it to be defined.